src/sentry/templates/sentry/emails/missing-members-nudge.html

```diff
@@ -9,10 +9,13 @@
     .card-container {
         display: flex;
         width: 100%;
-        margin-top: 16px;
-        margin-bottom: 20px;
+        justify-content: center;
+        margin: 16px -7px 20px -7px;
         padding: 0px;
-        gap: 14px;
+        > div {
+            margin-left: 7px;
+            margin-right: 7px;
+        }
```

Comment:
i'm also not sure these selectors work since styles are inlined

Reply:
they looked ok when i sent the emails locally, but i can be more explicit too

Reply:
Emails are tricky because some like gmail don't support a lot of css stuff. Make sure to test in gmail after deploying. For fly email I ended up implementing everything with table which was very painful 😭  Hope you're good.

Reply:
i forwarded them to myself and looked at them on gmail to fix some margin issues